From 652effa81ec5379b81e59ea04b5e443489f19dff Mon Sep 17 00:00:00 2001
From: Giovanni Ricca <giovanniricca@proton.me>
Date: Wed, 27 Aug 2025 15:56:53 +0200
Subject: [PATCH 3/9] sweet: powerhint: Drop TASchedtune{Boost, HighCap}

Reason: This is part of Google's sched ricing. We currently don't
include said chain on VantomKernel, so let's just remove it to clean up
our logs.

Change-Id: I53a55926d76599c9a3d29c934adec74f7109f3ae
---
 configs/power/powerhint.json | 30 ------------------------------
 1 file changed, 30 deletions(-)

diff --git a/configs/power/powerhint.json b/configs/power/powerhint.json
--- a/configs/power/powerhint.json
+++ b/configs/power/powerhint.json
@@ -101,24 +101,6 @@
       ],
       "ResetOnInit": true
     },
-    {
-      "Name": "TASchedtuneBoost",
-      "Path": "/dev/stune/top-app/schedtune.boost",
-      "Values": [
-        "40",
-        "10"
-      ],
-      "ResetOnInit": true
-    },
-    {
-      "Name": "TASchedtuneHighCap",
-      "Path": "/dev/stune/top-app/schedtune.prefer_high_cap",
-      "Values": [
-        "1",
-        "0"
-      ],
-      "ResetOnInit": true
-    },
     {
       "Name": "CPUBWHystTriggerCount",
       "Path": "/sys/class/devfreq/soc:qcom,cpu-cpu-llcc-bw/bw_hwmon/hyst_trigger_count",
@@ -325,18 +307,6 @@
       "Duration": 0,
       "Value": "1248000"
     },
-    {
-      "PowerHint": "INTERACTION",
-      "Node": "TASchedtuneBoost",
-      "Duration": 0,
-      "Value": "40"
-    },
-    {
-      "PowerHint": "INTERACTION",
-      "Node": "TASchedtuneHighCap",
-      "Duration": 0,
-      "Value": "1"
-    },
     {
       "PowerHint": "INTERACTION",
       "Node": "CPUBWHystTriggerCount",
-- 
2.53.0
